### PR TITLE
Improvements: Added new API to do PATCH request by fqn.

### PR DESCRIPTION
**Pull request by Siddhanttimeline**
### Describe your changes:
Added new API to do PATCH request by fqn.
PATCH path :/v1/{entity-name}/name/{fqn}

#
### Type of change:

- [ ] Bug fix
- [x] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-service/src/main/java/org/openmetadata/service/resources/apps/AppResource.java [hunk 1]
```diff
@@ -607,6 +607,50 @@ public Response patchApplication(
     return response;
   }
 
+  @PATCH
+  @Path("/fqn/{fqn}")
```
Comment (harshach) [inline openmetadata-service/src/main/java/org/openmetadata/service/resources/apps/AppResource.java:611]: we use /name eveywhere for fqn in other apis, lets keep that naming convention

Comment (Siddhanttimeline) [reply openmetadata-service/src/main/java/org/openmetadata/service/resources/apps/AppResource.java:611]: I will update the naming convention to use /name for fqn in the APIs and update the PR.

File: openmetadata-ui/src/main/resources/ui/.vscode/settings.json [hunk 2]
```diff
@@ -1,20 +1,18 @@
 {
-  // Use Prettier as default code formatter and enable formatting on save
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "editor.formatOnSave": true,
 
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true, // eslint --fix on save
-    "source.fixAll.stylelint": true, // stylelint --fix on save
-    "source.organizeImports": true // Organize imports on save
+    "source.fixAll.eslint": "explicit",
```
Comment (mohityadav766) [inline openmetadata-ui/src/main/resources/ui/.vscode/settings.json:6]: don't commit these changes @Siddhanttimeline

**Comment by Siddhanttimeline** [comment]
Adding test cases.

**Pull request closed without merging**